### ISSUE TITLE
Add a third site

### ISSUE THREAD

**Issue by hsivonen** [web-platform-tests/wpt#26412]
Currently, tests can depend on only two distinct sites: default and `alt`. Please add a third one for testing more complex frame hierarchies where each level is process-isolated from the others.

**Comment by stephenmcgruer** [comment]
This seems like a reasonable request to me. I don't think it would require an RFC? (cc @jgraham)

I think the very high level steps we'd need to do would be at least:

1. Add a new `alt2` site concept, most importantly updating the default in `tools/serve/serve.py`. (`also-not-web-platform.test` ?)
2. Plumb `alt2` through the `sub` code (and anywhere else `alt` is supported).
3. Update all of the certs/etc to also self-sign for the new site.

**Comment by jgraham** [comment]
I agree this is reasonable, but it seems like the kind of thing that does require an RFC, not because it's making backwards-incompatible changes but because it's user-facing API design (one might hope for the RFC to get feedback on whether the design addresses use cases people might have other than the specific one that motivated it). It also provides an oppertunity for people with different kinds of deployments to be informed about the change (wpt.live seems like a very relevant case here since it will require registering a new domain).

I think I agree with the high level steps for updating the default test environment. I really dislike `alt2` as a name ;)

**Comment by stephenmcgruer** [comment]
Good point jgraham - I stand corrected and an RFC should be required, yes.

Unfortunately my team on the Chromium side is currently spread thin, so this won't be a priority for us currently. If you feel ambitious @hsivonen , feel free to file an PR to https://github.com/web-platform-tests/rfcs :). These don't have to be very big (e.g. see https://github.com/web-platform-tests/rfcs/pull/67 as a recent example), just briefly laying out the idea (Summary), any interesting details (Details), and a consideration of any risks (Risks). For this, the risks are minor I think - mostly to downstream users of WPT who may have to change their infra to support another site.
